src/control/server/storage/scm/ipmctl.go

```diff
@@ -312,19 +318,18 @@ func (cr *cmdRunner) UpdateFirmware(deviceUID string, firmwarePath string) error
 // FIXME DAOS-10173: When libipmctl nvm_get_region() is fixed so that it doesn't take
 //                   a minute to return, replace this with getRegionState() below to use
 //                   libipmctl directly through bindings.
-func parseShowRegionOutput(log logging.Logger, text string) (_ storage.ScmState, err error) {
+func parseShowRegionOutput(log logging.Logger, text string) (_ []uint64, err error) {
```

Comment:
This seems a little strange. Why use a named return value for the error but not the slice? It's not wrong, exactly, but it was confusing and I had to scan through the rest of the function to figure out what was going on.

Reply:
done